wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -136,6 +136,61 @@ body.sensei {
 			border: solid 1px var(--sensei-secondary-color);
 		}
 	}
+
+	section.wp-block-sensei-lms-course-results {
+		display: flex;
+		flex-direction: column;
+		margin-top: 40px;
+		margin-bottom: 40px;
```

Comment:
How does `var(--wp--preset--spacing--30)` look? I generally use the closest theme var I can find unless it's a really specific size.

Reply:
I personally prefer to align with the design spec. I think that sometimes what I find acceptable, thinking it's just a small difference, might not be viewed the same way by others. Or in some cases, as you mentioned, we can use predefined styles due to the small difference, but in other cases, we use exact pixel values. This already leads to inconsistent styles. I feel that both ways more or less cause some confusion to some extent.
Ultimately, it's best to consult with the design team from the beginning to see if we can use predefined values. Based on previous experience, it's usually acceptable. That said, as I understand it, the design spec should align with our predefined styles? There shouldn't be numbers like 40px because, from what I've seen, the color, font-size, etc., all match up.

Reply:
> There shouldn't be numbers like 40px because, from what I've seen, the color, font-size, etc., all match up.

This is the key bit for me. That's why it has to be very exceptional for me to use a custom value for this kind of spacing. If the difference gets called out in a design review, that's when I'll start the discussion on changing the value in the system.